src/idealized_setup.jl

```diff
@@ -65,10 +65,11 @@ function idealized_setup(arch;
     
     # # Cooling in the middle of the domain and heating outside?
     # @inline Qtop(x, y, t, p) = - p.Q / p.ρ₀ / p.cₚ * cos(2π * x / p.Lx)
+    @inline Qtop(x, y, t) = Q / ρ₀ / cₚ * (1 - tanh((t - 10days) / 12hours)) / 2
 
     u_top = FluxBoundaryCondition(τw * cosd(θ) / ρ₀)
     v_top = FluxBoundaryCondition(τw * sind(θ) / ρ₀)
-    T_top = FluxBoundaryCondition(Q / ρ₀ / cₚ) # Positive fluxes at the top are cooling in Oceananigans
+    T_top = FluxBoundaryCondition(Qtop)# / ρ₀ / cₚ) # Positive fluxes at the top are cooling in Oceananigans
```

Comment:
Why changing this? Are you prescribing fluxes of temperature instead of heat?

Reply:
It's testing a time dependent temperature flux Qtop defined above to stop the flux after 10 days.